src/app/swap/swap-out.ts

```diff
@@ -0,0 +1,69 @@
+import { BTC_NETWORK, getColdStorageConfig, getSwapConfig } from "@config"
+import { TxDecoder } from "@domain/bitcoin/onchain"
+import { SwapServiceError } from "@domain/swap/errors"
+import { OnChainService } from "@services/lnd/onchain-service"
+import { toSats } from "@domain/bitcoin"
+import { SwapOutChecker } from "@domain/swap"
+import { baseLogger } from "@services/logger"
+import { SwapService } from "@services/swap"
+import { getActiveLnd } from "@services/lnd/utils"
+import { getChannelBalance } from "lightning"
+import { addAttributesToCurrentSpan } from "@services/tracing"
+
+const logger = baseLogger.child({ module: "swap" })
+
+export const swapOut = async (
+  amount: Satoshis,
+): Promise<SwapOutResult | SwapServiceError> => {
+  const swapService = SwapService()
+  logger.info("SwapApp: Started")
+  const onChainService = OnChainService(TxDecoder(BTC_NETWORK))
+  if (onChainService instanceof Error) return onChainService
+
+  const onChainBalance = await onChainService.getBalance()
+  if (onChainBalance instanceof Error) return onChainBalance
+  const outbound = await getChannelLiquidityBalance()
+  const minOnChainHotWalletBalanceConfig =
+    getColdStorageConfig().minOnChainHotWalletBalance
+  const minOutboundLiquidityBalance = getSwapConfig().minOutboundLiquidityBalance
+
+  const swapChecker = SwapOutChecker({
+    currentOnChainHotWalletBalance: onChainBalance,
+    minOnChainHotWalletBalanceConfig,
+    currentOutboundLiquidityBalance: outbound,
+    minOutboundLiquidityBalance,
+  })
+  const isOnChainWalletDepleted = swapChecker.isOnChainWalletDepleted()
+  const isOutboundLiquidityDepleted = swapChecker.isOutboundLiquidityDepleted()
+  logger.info(`SwapApp: isOnChainWalletDepleted: ${isOnChainWalletDepleted},
+    isOutboundLiquidityDepleted: ${isOutboundLiquidityDepleted}`)
```

Comment:
```suggestion
  logger.info({isOnChainWalletDepleted, isOutboundLiquidityDepleted }, "wallet depletion status")
```

Reply:
our logger is a json logger. this make it easier to parse in the backend. see https://getpino.io/#/